examples/cannon/cannon.py

```diff
@@ -0,0 +1,127 @@
+from charm4py import charm, Chare, Array, Future, coro, Channel
+import time
+import numpy as np
+
+try:
+    from numba import njit
+except ImportError:
+    # create a dummy numba.njit decorator
+    def njit(func):
+        return func
+
+@njit
+def matmul(C, A, B):
+    C += A @ B
+
+class SubMatrix(Chare):
+    def __init__(self, subdim_size, charedim, init_done):
```

Comment:
I think this is missing a call to `super(SubMatrix, self).__init__(...)`, hence the LGTM warning.